org.scantailor.ScanTailor.json

```diff
@@ -0,0 +1,58 @@
+{
+  "app-id": "org.scantailor.ScanTailor",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
```

Comment:
Switch to `18.08` and restest.

Reply:
https://github.com/flathub/shared-modules/pull/35